### PR TITLE
Adds postcss-import plugin + fixes import for react phone styles

### PR DESCRIPTION
**Pull request by taylorjdawson**
### CSS Import Rules Error

After importing `import '@turnkey/sdk-react/styles';` in the `layout.tsx` file and running `npm run dev`, an error occurred. When bundling CSS with Rollup using `rollup-plugin-postcss`, `@import` rules are not being correctly placed at the top of the final CSS bundle. This violates CSS specifications, which require `@import` statements to precede all other rules (except `@charset` and `@layer`).

#### Solution

Add `postcss-import` plugin to the PostCSS configuration to the Rollup setup.

The `postcss-import` plugin processes `@import rules` and ensures they appear at the top of the CSS file. By including this in the PostCSS plugins array, all `@import` rules are handled correctly before other CSS rules are processed, preventing build errors.

```bash
@import rules must precede all rules aside from @charset and @layer statements at [project]/node_modules/@turnkey/sdk-react/dist/styles.esm.css:0:6309
```

#### Additional Build Error with postcss-import

When using `postcss-import`, I encountered a build error related to importing CSS from external packages:

```bash
postcss-import: Failed to find 'react-international-phone/style.css'
```

This occurs because `postcss-import` cannot resolve the CSS import from the `node_modules` directory when it's imported within a CSS file.

##### Solution

Instead of importing the styles in `PhoneInput.css`:

```css
@import "react-international-phone/style.css";
```

Move the import to the React component file (`PhoneInput.tsx`) directly:

```tsx
import "react-international-phone/style.css";
```

This approach allows the bundler to correctly resolve the CSS import from `node_modules`, fixing the build error.

## Did you add a changeset?

If updating one of our packages, you'll likely need to add a changeset to your PR. To do so, run [`pnpm changeset`](https://pnpm.io/using-changesets#adding-new-changesets). `pnpm changeset` will generate a file where you should write a human friendly message about the changes. Note how this ([example](https://github.com/tkhq/sdk/blob/b409cd06790f011bf939adcf0755499b8e7497ae/.changeset/extra-http-exports.md?plain=1#L1)) includes the package name (should be auto added by the command) along with the type of [semver change (major.minor.patch)](https://semver.org/) (which you should set).

These changes will be used at release time to determine what packages to publish and how to bump their version. For more context see [this comment](https://github.com/tkhq/sdk/pull/67#issuecomment-1568838440).

### CODE REVIEW

File: packages/sdk-react/src/components/auth/Auth.tsx [hunk 1]
```diff
@@ -60,7 +64,7 @@ interface AuthProps {
     facebookEnabled: boolean;
     googleEnabled: boolean;
   };
-  configOrder: string[];
+  configOrder: AuthSection[];
```
Comment (moe-dev) [inline packages/sdk-react/src/components/auth/Auth.tsx:67]: let's make these changes in another PR - this breaks example build since this type is not exported and will not be consistent with our docs at the moment

**Review by moe-dev — APPROVED** [review]

**Review by turnekybc — APPROVED** [review]

File: packages/sdk-react/rollup.config.mjs [hunk 2]
```diff
@@ -1,47 +1,54 @@
-import typescript from "@rollup/plugin-typescript";
-import nodeExternals from "rollup-plugin-node-externals";
-import path from "node:path";
-import postcss from "rollup-plugin-postcss";
-import preserveDirectives from "rollup-preserve-directives";
-import url from "@rollup/plugin-url";
-import alias from "@rollup/plugin-alias";
-import copy from "rollup-plugin-copy";
+import typescript from '@rollup/plugin-typescript';
```
Comment (andrewkmin) [inline packages/sdk-react/rollup.config.mjs:1]: would expect these to be double quotes

Comment (moe-dev) [reply packages/sdk-react/rollup.config.mjs:1]: from all the package READMES it shows single quotes actually. It seems like we were incorrect before
